test/registry/TestReverseRegistrar.js

```diff
@@ -1,56 +1,82 @@
-
+const namehash = require('eth-ens-namehash')
+const sha3 = require('web3-utils').sha3
 const PublicResolver = artifacts.require('./resolvers/PublicResolver.sol');
 const ReverseRegistrar = artifacts.require('./registry/ReverseRegistrar.sol');
 const ENS = artifacts.require('./registry/ENSRegistry.sol');
 const NameWrapper = artifacts.require('DummyNameWrapper.sol');
+const { exceptions } = require("../test-utils")
 
-const namehash = require('eth-ens-namehash');
-const sha3 = require('web3-utils').sha3;
+describe('ReverseRegistrar Tests', () => {
 
 contract('ReverseRegistar', function (accounts) {
 
-    let node;
+    let node, node2, node3;
+    
     let registrar, resolver, ens, nameWrapper;
 
     beforeEach(async () => {
         node = namehash.hash(accounts[0].slice(2).toLowerCase() + ".addr.reverse");
+        node2 = namehash.hash(accounts[1].slice(2).toLowerCase() + '.addr.reverse');
+        node3 = namehash.hash(accounts[2].slice(2).toLowerCase() + '.addr.reverse');
         ens = await ENS.new();
         nameWrapper = await NameWrapper.new();
         resolver = await PublicResolver.new(ens.address, nameWrapper.address);
         registrar = await ReverseRegistrar.new(ens.address, resolver.address);
+        await registrar.setController(accounts[0], true)
```

Comment:
Took a while to figure out this is the key logic. You may either want to have this logic in each test where it matters to make it explicit.